doc/release/README.chplenv.rst

```diff
@@ -0,0 +1,555 @@
+       re2     if the build was successful
+       none    otherwise
+       ======= ===============================
+
+   .. note::
+     Note that the Chapel util/quickstart/setchplenv.* source scripts set
+     CHPL_REGEXP to 'none' while the util/setchplenv.* versions leave it unset,
+     resulting in the behavior described just above.
+
+
+CHPL_AUX_FILESYS
+~~~~~~~~~~~~~~~~
+   Optionally, the CHPL_AUX_FILESYS environment variable can be used to
+   request that runtime support for filesystems beyond the usual Linux one be
+   present.  Current options are:
+
+       ====== =================================================
+       Value  Description
+       ====== =================================================
+       none   only support traditional Linux filesystems
+       hdfs   also support HDFS filesystems
+       curl   also support CURL as a filesystem interface
+       ====== =================================================
+
+   If unset, CHPL_AUX_FILESYS defaults to "none".
+
+   See $CHPL_HOME/doc/technotes/README.hdfs for more information about
+   HDFS support.
+   See $CHPL_HOME/doc/technotes/README.curl for more information about
+   CURL support.
+
+
+CHPL_LLVM
+~~~~~~~~~
+   Optionally, the CHPL_LLVM environment variable can be used to
+   enable support for the LLVM back-end to the Chapel compiler (see
+   doc/technotes/README.llvm) or to support extern blocks in Chapel code via
+   the Clang compiler (see doc/technotes/README.extern).  Current options are:
+
+       ====== ======================================================
+       Value  Description
+       ====== ======================================================
+       llvm   use the llvm/clang distribution in third-party
+       none   do not support llvm-/clang-related features
+       ====== ======================================================
+
+   If unset, CHPL_LLVM defaults to "llvm" if you've already installed llvm in
+   third-party and "none" otherwise.
+
+
+CHPL_WIDE_POINTERS
+~~~~~~~~~~~~~~~~~~
+   Optionally, the CHPL_WIDE_POINTERS environment variable can be used to
+   specify the wide pointer format for multilocale programs.  Current options
+   are:
+
+       ======== =============================================================
+       Value    Description
+       ======== =============================================================
+       struct   store wide pointers in structures which may span more than
+                one word
+       nodeN    ("N" a number, 2 <= N <= 60) store wide pointers in single
+                words, with N bits used to store the node (top level locale)
+                number and the rest containing the address on that node
+       ======== =============================================================
+
+   CHPL_WIDE_POINTERS is used to select between two modes of operation.  One is
+   universally applicable; the other has restricted applicability but may
+   reduce remote communication.
```

Comment:
FWIW I think we should just focus on the RST conversion for this PR, and discuss content in another once an owner is found for content.